packages/component-base/src/virtualizer-iron-list-adapter.js

```diff
@@ -797,7 +797,7 @@ export class IronListAdapter {
       this.__skipNextVirtualIndexAdjust = false;
     } else if (Math.abs(delta) > 10000) {
       // Process a large scroll position change
-      const scale = this._scrollTop / (this.scrollTarget.scrollHeight - this.scrollTarget.offsetHeight);
+      const scale = this._scrollTop / (this.scrollTarget.scrollHeight - this.scrollTarget.clientHeight);
```

Comment:
**note:** `offsetHeight` includes the height of borders, paddings and scrollbars whereas `scrollHeight` does not. This was the root of the calculation error.